code/modules/jobs/jobs.dm

```diff
@@ -70,7 +70,7 @@ GLOBAL_LIST_INIT(nonhuman_positions, list(
 GLOBAL_LIST_INIT(munitions_positions, list(
 	"Master At Arms",
 	"Munitions Technician",
-	"CAG",
+	"Flight Leader",

```

Comment:
"Air group commander"?
"Attack wing lead"
Flight leader sounds OK, but if we're changing it from CAG I want to make it 100% clear what their job is

Reply:
Agreed. Workshopping required.